File: BusinessLogic/src/models/public/Dog.cs
namespace BusinessLogic.src.models
{
    public class Dog : Pet
    {
        public string name { get; set; }
        public string address { get; set; }
        public string owner { get; set; }
        public string breed { get; set; }
        public string breeder { get; set; }
        public bool outdoor { get; set; }
        public bool attack { get; set; }
        public bool service { get; set; }
        public int height { get; set; }
        public int weight { get; set; }
    }
}

File: PetShop/src/app/core/models/forms/pets/dog.ts
import { Pet } from './pet';

export class Dog implements Pet {
  name: string = null;
  address: string = null;
  owner: string = null;
  breed: string = null;
  breeder: string = null;
  outdoor: boolean = false;
  attack: boolean = false;
  service: boolean = false;
  height: number = null;
  weight: number = null;
}
